extensions/transferdetails/service.go

```diff
@@ -0,0 +1,284 @@
+package transferdetails
+
+import (
+	"context"
+	"reflect"
+
+	"github.com/centrifuge/go-centrifuge/documents"
+	"github.com/centrifuge/go-centrifuge/errors"
+	"github.com/centrifuge/go-centrifuge/extensions"
+	"github.com/centrifuge/go-centrifuge/identity"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	logging "github.com/ipfs/go-log"
+)
+
+// Service defines specific functions for extension funding
+type Service interface {
+	documents.Service
```

Comment:
We should change it do:
```suggestion
	srv documents.Service
```

We don't want to expose the general service method to the new handlers.
We need to change it for funding agreement later too.

Reply:
I understand what you mean, but it is not possible here as this is an interface, not a struct